src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout_19h1_themeresources.xaml

```diff
@@ -0,0 +1,273 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
```

Comment:
All these _themeresources  files are necessary?

Reply:
Yes, it's part of the WinUI Import. It'll be easier to track updates made in the original repository.